File: core-java-practice/gcr-codebase/programming-elements/PowerCalculation.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int base = sc.nextInt();
        int power = sc.nextInt();

        double result = Math.pow(base, power);

        System.out.println(result);
    }
}

File: core-csharp-practice/gcr-codebase/programming-elements/PowerCalculation.cs
using System;

class PowerCalculation {
    public static void Main(String[] args) {
        int baseN = Convert.ToInt32(Console.ReadLine());
        int power = Convert.ToInt32(Console.ReadLine());

        double result = Math.Pow(baseN, power);

        Console.WriteLine(result);
    }
}